activesupport/lib/active_support/lazy_load_hooks.rb

```diff
@@ -28,6 +31,15 @@ def self.extended(base) # :nodoc:
       end
     end
 
+    # Declares a method that makes it possible to extend components of Rails
```

Comment:
This method is not only used in rails so we should make the documentation more generic